client/src/actions/proposition/propositionsActions.js

```diff
@@ -1,49 +1,58 @@
 import axios from 'axios'
-import base64 from 'base-64'
 import {
   FETCH_PROPOSITIONS_REQUEST,
   FETCH_PROPOSITIONS_SUCCESS,
-  FETCH_PROPOSITIONS_FAILURE,
-} from '../../constants/ActionTypes'
-
+  FETCH_PROPOSITIONS_FAILURE
+} from "../../constants/ActionTypes";
 
 const fetchPropositionsRequest = () => {
   return {
     type: FETCH_PROPOSITIONS_REQUEST
-  }
-}
+  };
+};
 
 const fetchPropositionsSucess = propositions => {
-    return {
-        type: FETCH_PROPOSITIONS_SUCCESS,
-        payload: propositions.data
-    }
-}
+  return {
+    type: FETCH_PROPOSITIONS_SUCCESS,
+    payload: propositions.data
+  };
+};
 
 const fetchPropositionsFailure = error => {
-    return {
-        type: FETCH_PROPOSITIONS_FAILURE,
-        payload: error
-    }
-}
+  return {
+    type: FETCH_PROPOSITIONS_FAILURE,
+    payload: error
+  };
+};
 
 export const fetchPropositionsGame = (idUsr, idChallenge) => {
   return async function(dispatch, getState) {
-    if (getState().propositions.data.length === 0) {
-      dispatch(fetchPropositionsRequest())
-      let username = getState().user.currentUser.username;
-      axios.post('http://localhost:55555/users/'+idUsr+'/challenge/'+idChallenge+'/propsitions', null, {
-        headers: {
-          Authorization: "Basic" + base64.encode(username + ":" +localStorage.getItem("password"))
+    dispatch(fetchPropositionsRequest());
+    let base64 = require("base-64");
+    let username = getState().user.currentUser.username;
+    axios
+      .post(
+        "http://localhost:55555/users/" +
```

Comment:
instead localhost could we do another commit to use `process.env.REACT_APP_API_HOST` and you can configure it when needed using the `.env` file

Reply:
Ready